react/utils/initialSearchUtils.js

```diff
@@ -0,0 +1,33 @@
+import { PATH_SEPARATOR, MAP_VALUES_SEP } from '../constants'
+
+export const isInitialFacet = (facet, initialSearch) => {
+  if (!initialSearch) {
+    return false
+  }
+
+  const initialFacets = initialSearch.specificationFilters.flatMap(filter =>
```

Comment:
Do we have a polyfill to `flatMap` in the store? Do you people know anything about that @thalytafabrine @vtex-apps/store-framework-devs ?

Reply:
we don't, but we can add it [here](https://github.com/vtex/render-server/blob/master/node/utils/assetLoader.ts#L35) as `Array.prototype.flatMap`

Reply:
@Klynger I opened a PR with this: https://github.com/vtex/render-server/pull/693 😅

Reply:
Great! LGTM